example/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,9 @@
             </intent-filter>
         </activity>
 
-        <activity android:name=".gallery.GalleryActivity" />
+        <activity android:name="nl.nos.imagin.example.gallery_vp1.GalleryActivity" />
```

Comment:
Why would you keep this GalleryActivity?

Reply:
No reason! I'll revert this